sdk/formrecognizer/azure-ai-formrecognizer/tests/test_custom_forms_async.py

```diff
@@ -224,6 +224,11 @@ def callback(raw_response, _, headers):
         self.assertIsNotNone(recognized_form[0].model_id)
         self.assertUnlabeledFormFieldDictTransformCorrect(recognized_form[0].fields, actual_fields, read_results)
 
+        recognized_form_dict = [v.to_dict() for v in recognized_form]
+        self.assertIsNone(recognized_form_dict[0].get("form_type_confidence"))
+        self.assertIsNotNone(recognized_form_dict[0].get("model_id"))
+        self.assertEqual(recognized_form_dict[0].get("form_type"), "form-0")
```

Comment:
would you mind adding a similar test for one that returns CustomFormModel? I think that should cover the more complex models.